src/nft/components/explore/Banner.tsx

```diff
@@ -68,7 +70,11 @@ const Banner = () => {
   const { data: collections } = useQuery(
     ['trendingCollections'],
     () => {
-      return fetchTrendingCollections({ volumeType: 'eth', timePeriod: TimePeriod.OneDay, size: 5 })
+      return fetchTrendingCollections({
+        volumeType: 'eth',
+        timePeriod: TimePeriod.OneDay,
+        size: TRENDING_COLLECTION_SIZE + LOADING_BUFFER,
```

Comment:
what's the purpose of the +5 increase here? Is it just to make it more likely there's a renderable collection?

Reply:
left overs from the previous wip take on dynamic filtering (where @cbachmeier suggested we fetch like 20-25 items). With hardcoded addresses, we'll just extend by the length of the excluded array.